EasyPost.Tests/ReportTest.cs

```diff
@@ -27,11 +26,20 @@ private static async Task<Report> CreateBasicReport(string reportType)
             });
         }
 
-        private static async Task TestCreateReport(string cassetteName, string reportType, string idPrefix)
+        private static async Task<Report> CreateAdvancedReport(string reportType, Dictionary<string, object> parameters)
         {
-            VCR.Replay(cassetteName);
+            parameters["start_date"] = Fixture.ReportStartDate;
+            parameters["end_date"] = Fixture.ReportEndDate;
+            return await Report.Create(reportType, parameters);
+        }
 
-            Report report = await CreateBasicReport(reportType);
+#nullable enable
+        private static async Task TestCreateReport(string cassetteName, string reportType, string idPrefix, Dictionary<string, object>? parameters = null)
+        {
+            VCR.Replay(cassetteName);
+            Report report = parameters == null
+                ? await CreateBasicReport(reportType)
+                : await CreateAdvancedReport(reportType, parameters);
```

Comment:
This test will only ever test a basic report creation (no columns) so why have the conditional advanced params here?

Reply:
Because :)